src/sidebar/components/annotation.js

```diff
@@ -11,27 +11,6 @@ var isNew = annotationMetadata.isNew;
 var isReply = annotationMetadata.isReply;
 var isPageNote = annotationMetadata.isPageNote;
 
-/** Return a human-readable error message for the given server error.
- *
- * @param {object} reason The error object from the server. Should have
- * `status` and, if `status` is not `0`, `statusText` and (optionally)
- * `data.reason` properties.
- *
- * @returns {string}
- */
-function errorMessage(reason) {
-  var message;
-  if (reason.status <= 0) {
-    message = 'Service unreachable.';
-  } else {
-    message = reason.status + ' ' + reason.statusText;
-    if (reason.data && reason.data.reason) {
-      message = message + ': ' + reason.data.reason;
-    }
-  }
-  return message;
-}
```

Comment:
I'm deleting these tests because they are unnecessary now that the logic for testing translation of API errors into messages is done in `store-test.js`. The `<annotation>` component now only needs a test that it displays any such messages.